Main.cs

```diff
@@ -323,6 +323,30 @@ public List<Result> LoadContextMenus(Result selectedResult)
             }
 
             var icoPath = (record.Type == ResultType.File) ? "Images\\file.png" : "Images\\folder.png";
+
+            if (_settings.ShowShellContextMenu)
+            {
+	            contextMenus.Add(new Result
+	            {
+		            Title = _context.API.GetTranslation("flowlauncher_plugin_everything_show_shell_context_menu"),
+		            Action = (context) =>
+		            {
+			            var fileInfos = new FileInfo[] { new(record.FullPath) };
+
+			            var showPosition = new System.Drawing.Point
+			            (
+				            System.Windows.Forms.Screen.PrimaryScreen.WorkingArea.Width / 2,
```

Comment:
Is dividing by two just a way to get the window to be centred? Is there a better way to do this? If not could we add a quick comment as to why by 2 please.

Reply:
My intent is to get the Windows Context Menu to be centered. I've extracted x, y of the point as variables instead of a quick comment.

Additionally, I've made some changes to show Windows Context Menu on the screen that has the mouse cursor for the case of multi-monitor users.